### DISCUSSION TITLE
Sea ice?

### DISCUSSION THREAD

**Discussion by francispoulin** [CliMA/Oceananigans.jl#2878] [Ideas]
I remember last year there was discussion of adding sea ice into the repo.  I realize this is a big task but wondered if anyone was working on this or had thoughts as to how this should begin?  

I am happy to chat on Zoom if that's easier?

**Comment by glwagner** [comment]
Sea ice model development is definitely planned and we may put out a call to hire someone for that soon!

I would like the sea ice model code to live in different repo.

Here's a prototype:

https://github.com/glwagner/IcyAntics.jl

**Reply by francispoulin** [reply]
Thanks @glwagner , that looks like a great foundation for the sea ice mode! 

I might try playing with this sometime soon and I would be happy to help with some parts of this in the future.
